### PR TITLE
Make Round Work

### PR DESCRIPTION
**Pull request by gl-lei**
Make the time work
Make the round work
Make the totalTime and the time work together

### CODE REVIEW

File: GymWorkoutManager/FirstViewController.swift [hunk 1]
```diff
@@ -17,26 +17,26 @@ class FirstViewController: UIViewController, TimeSetupViewControllerDelegate {
...
 
     
     @IBAction func counter(sender: AnyObject) {
         if startButton.currentTitle != "Stop" {
             startButton.setTitle("Stop", forState: .Normal)
-            timerCountdown = NSTimer.scheduledTimerWithTimeInterval(0.01, target: self, selector: Selector("timeCountdown"), userInfo: nil, repeats: true)
-            timerRuning = NSTimer.scheduledTimerWithTimeInterval(0.01, target: self, selector: Selector("timeRuning"), userInfo: nil, repeats: true)
-            startRuningTime = NSDate.timeIntervalSinceReferenceDate()
+            
+            self.timerCountdown = NSTimer.scheduledTimerWithTimeInterval(0.01, target: self, selector: Selector("timeCountdown"), userInfo: nil, repeats: true)
```
Comment (gl-lei) [inline GymWorkoutManager/FirstViewController.swift:34]: 使用同一个计时器

File: GymWorkoutManager/FirstViewController.swift [hunk 2]
```diff
@@ -55,14 +55,33 @@ class FirstViewController: UIViewController, TimeSetupViewControllerDelegate {
...
+            self.repeatTimer.text = self.timeString(self.time).stringByReplacingOccurrencesOfString(".", withString: ":")
+            self.totalWorkoutTimer.text = self.timeString(self.totalTime).stringByReplacingOccurrencesOfString(".", withString: ":")
+        }
     }
     
     // MARK: - View
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        self.time = self.timeDate(self.receivedTime)
```
Comment (gl-lei) [inline GymWorkoutManager/FirstViewController.swift:83]: 设置初始值

File: GymWorkoutManager/FirstViewController.swift [hunk 3]
```diff
@@ -74,9 +93,13 @@ class FirstViewController: UIViewController, TimeSetupViewControllerDelegate {
     
     func timeSetupFinish(timeSetupViewController: TimeSetupViewController, result: [String]) {
         self.receivedTime = result
+        self.round = result[2]
         self.time = self.timeDate(result)
+        self.totalTime = self.timeDate(["0", "0", "0"])
+        
         self.repeatTimer.text = self.timeString(result).stringByReplacingOccurrencesOfString(".", withString: ":")
-        self.aroundNumber.text = result[2]
+        self.totalWorkoutTimer.text = self.timeString(self.totalTime).stringByReplacingOccurrencesOfString(".", withString: ":")
```
Comment (gl-lei) [inline GymWorkoutManager/FirstViewController.swift:101]: 每次返回，将总时间重置为00:00:00

File: GymWorkoutManager/FirstViewController.swift (hunk 1 above)
Comment (iceman201) [inline GymWorkoutManager/FirstViewController.swift:34]: :+1:

File: GymWorkoutManager/FirstViewController.swift (hunk 2 above)
Comment (iceman201) [inline GymWorkoutManager/FirstViewController.swift:83]: :+1:

File: GymWorkoutManager/FirstViewController.swift (hunk 3 above)
Comment (iceman201) [inline GymWorkoutManager/FirstViewController.swift:101]: :+1:

**Pull request merged**